proto/provenance/metadata/v1/specification.proto

```diff
@@ -91,6 +100,22 @@ message ContractSpec {
   repeated ConsiderationSpec consideration_specs = 5;
 }
 
+// Info holds general information that is handy to associate with a structure.
+message Info {
+  // A Name for this thing.
+  string name = 1;
+  // A description of this thing.
+  string description = 3;
+  // URL to find even more info.
+  string website_url = 4 [
+    (gogoproto.moretags)   = "yaml:\"website_url\""
+  ];
+  // URL of an icon for this. thing.
```

Comment:
nit: Extra period.  Could be a bit more clear.

Reply:
Fixed -> "// URL of an icon."